### PR TITLE
[fix/remove-nft] remove NFT tab

### PR DESCRIPTION
**Pull request by ThanhNguyen140797**
### Summary 

Remove NFT Tab

### Checklist

-   [x] I removed all commented or unnecessary code.
-   [ ] Provide the screenshots due to UI changed or bug fixed
-   [ ] My code has covered these test cases (please list down your tested cases):

### CODE REVIEW

**Review by djhahe — APPROVED** [review]

File: src/screens/authentication/data/data.tsx [hunk 1]
```diff
@@ -16,11 +16,12 @@ export interface NFT {
   time: string;
 }
 const ListIntro: Array<Intro> = [
-  {
-    id: 0,
-    image: images.intro1,
-    title: 'The new NFT marketplace',
-  },
+  // TODO: enable in the next version.
```
Comment (gaupoit) [inline src/screens/authentication/data/data.tsx:19]: hi @ThanhNguyen140797 @djhahe , please help me to review again. We need to hide the NFT screen in welcome screen alos.

File: src/screens/authentication/WelcomeScreen/index.tsx [hunk 2]
```diff
@@ -130,7 +130,7 @@ function WelcomeScreen() {
         </CButton>
         <CPaginationDot
           lineStyle
-          length={3}
+          length={arrayImgSlider.length}
```
Comment (gaupoit) [inline src/screens/authentication/WelcomeScreen/index.tsx:133]: Fix magic number which is length of sliders.

File: src/screens/authentication/WelcomeScreen/index.tsx [hunk 3]
```diff
@@ -144,7 +144,7 @@ function WelcomeScreen() {
         />
       </Row.LR>
       <Col style={[styles.footer, { bottom: insets.bottom }]}>
-        {currentIndex === 2 ? _renderGetStarted() : _renderNavigatorButton()}
+        {currentIndex === arrayImgSlider.length - 1 ? _renderGetStarted() : _renderNavigatorButton()}
```
Comment (gaupoit) [inline src/screens/authentication/WelcomeScreen/index.tsx:147]: Fix magic number which is the index of last item in array of sliders.

File: src/screens/authentication/data/data.tsx (hunk 1 above)
Comment (ThanhNguyen140797) [reply src/screens/authentication/data/data.tsx:19]: LGTM

**Pull request merged**